### PR TITLE
[8.0][MIG] Backport of partner_coc

### PR DESCRIPTION
**Pull request by pankk**
Backport of the 10.0 module partner_coc.

Note that this depends on #612.

### CODE REVIEW

**Review by tarteo — CHANGES_REQUESTED** [review]
I get an error:
`AttributeError: 'res.partner' object has no attribute '_compute_identification'` 
I think the method _compute_identification is not available in 8.0. I think you have to manually create res.partner.id_number

**Comment by pankk** [comment]
@tarteo Thanks for testing! I mixed up two versions of partner_identification. The one on the 8.0 branch in this repo is a backport of 9.0. I was using a version that was backported from the 10.0 version. So I should test if the 10.0 backport doesn't break compatibility with anything that depends on it (i.e. partner_identification_gln) if we want to use that version. I also need to confirm who did the backport so they can be credited before I (or they) make a PR for it.

**Comment by pankk** [comment]
I've since added #612 to solve the issue @tarteo reported above.

**Comment by rafaelbn** [comment]
===========
Partner CoC
===========

Adds a field 'Chamber Of Commerce Registration Number' to partner.

**Comment by thomaspaulb** [comment]
@rafaelbn I dont see that missing `=` anymore; time for an updated review from your side?

**Review by thomaspaulb — APPROVED** [review]
PyLint and Travis errors are unrelated.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Review by rafaelbn — APPROVED** [review]
:+1:

File: partner_coc/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner CoC',
+    'summary': "Adds a field 'Chamber Of Commerce Registration Number' to "
+               "partner",
+    'version': '8.0.1.0.0',
+    'category': 'Web',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'http://www.onestein.eu',
```
Comment (NL66278) [inline partner_coc/__openerp__.py]: Change to https://github.com/OCA/partner-contact

File: partner_coc/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (NL66278) [inline partner_coc/__manifest__.py]: Delete this file

**Review by NL66278 — CHANGES_REQUESTED** [review]
Needs a few changes. See other comments.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**